micromamba/tests/test_list.py

```diff
@@ -28,6 +31,51 @@ def test_list(tmp_home, tmp_root_prefix, tmp_env_name, tmp_xtensor_env, env_sele
         for i in res
     )
 
+    if reverse_flag == "--reverse":
+        assert names.index("xtensor") > names.index("xtl")
+    else:
+        assert names.index("xtensor") < names.index("xtl")
+
+
+@pytest.mark.parametrize("reverse_flag", ["", "--reverse"])
+@pytest.mark.parametrize("quiet_flag", ["", "-q", "--quiet"])
+@pytest.mark.parametrize("env_selector", ["", "name", "prefix"])
+@pytest.mark.parametrize("shared_pkgs_dirs", [True], indirect=True)
+def test_list_no_json(
+    tmp_home, tmp_root_prefix, tmp_env_name, tmp_xtensor_env, env_selector, quiet_flag, reverse_flag
+):
+    if env_selector == "prefix":
+        res = helpers.umamba_list("-p", tmp_xtensor_env, quiet_flag, reverse_flag)
+    elif env_selector == "name":
+        res = helpers.umamba_list("-n", tmp_env_name, quiet_flag, reverse_flag)
+    else:
+        res = helpers.umamba_list(quiet_flag, reverse_flag)
+
+    assert len(res) > 10
+
+    assert "xtensor" in res
+    assert "xtl" in res
+
+    # This is what res looks like in this case (with or without a header delimiter):
+    # List of packages in environment: "xxx"
+
+    # Name           Version  Build        Channel
+    # ────────────────────────────────────────────────────
+    # _libgcc_mutex  0.1      conda_forge  conda-forge
+    # _openmp_mutex  4.5      2_gnu        conda-forge
+    packages = res[res.rindex("Channel") :].split("\n", 1)[1]
+    packages_list = packages.strip().split("\n")
+    header_delimiter = "\u2500"
```

Comment:
Are there problems with this:
```suggestion
    header_delimiter = "─"
```
?

Reply:
I am not sure which of the different dashes this one is... I will have to check !

Reply:
FYI, I copied and pasted it from the output of the failing test.